float8_experimental/fsdp_utils.py

```diff
@@ -81,6 +81,8 @@ def precompute_float8_dynamic_scale_for_fsdp(module: nn.Module) -> None:
     torch.ops.aten.as_strided.default,
     torch.ops.aten._to_copy.default,
     torch.ops.aten._pin_memory.default,
+    torch.ops.aten.split.Tensor,
```

Comment:
`aten.split` is from `torch.chunk`, when calling from `distribute_tensor` during TP init

@awgu curious if you still remember the reason to return bf16 from torch.chunk. Is it for padding? any concerns if I prefer `torch.chunk` to returning `WeightWithDynamicFloat8CastTensor` ?

Reply:
> @awgu curious if you still remember the reason to return bf16 from torch.chunk.

I thought that dtype and whether is `WeightWithDynamicFloat8CastTensor` are orthogonal. Do you mean the latter (whether is `WeightWithDynamicFloat8CastTensor` or not?

I think originally I only added the ops that I saw I needed. Adding `aten.split` and `aten.clone` seems okay to me.

Reply:
> whether is WeightWithDynamicFloat8CastTensor or not

exactly, `WeightWithDynamicFloat8CastTensor` or not is the key. I edited my previous comments to say right now torch.chunk returns `Tensor`

> I think originally I only added the ops that I saw I needed

changing `torch.chunk` affects both TP and FSDP2. will double check FSDP2 after the change